src/main/resources/edu/cornell/kfs/ksr/document/datadictionary/KSRBaseTypes.xml

```diff
@@ -0,0 +1,10 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:p="http://www.springframework.org/schema/p"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+                http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+                
+    <bean id="PrimaryDepartmentCodePatternConstraint" abstract="true"
+          class="org.kuali.kfs.krad.datadictionary.validation.constraint.ConfigurationBasedRegexPatternConstraint"
+          p:labelKey="error.ksr.securityrequestdocument.primaryDeptCode.regex" p:patternTypeKey="ksr.securityrequestdocument.primaryDeptCode.regex"/>
```

Comment:
Have properties been added for this regex and label? I'm not seeing them defined in this PR.

Reply:
added now